site/styles/_meta-sub-item.scss

```diff
@@ -1,14 +1,23 @@
 .sub-item {
   display: grid;
-  grid-gap: 15px;
+  gap: 1px 15px;
   grid-template-columns: 112px minmax(100px, 1fr);
   grid-template-rows: auto minmax(0, 1fr);
-  margin-bottom: 15px;
-  padding-bottom: 15px;
+  margin-bottom: 16px;
+  padding-bottom: 16px;
```

Comment:
The GoogleDoc says these should add up to 40px between the bonus content and the recommendations slider... I only see 32px total.

Reply:
Its at 34px
![reply1](https://user-images.githubusercontent.com/32156362/200696836-4f4afffc-fd8d-4a84-863a-6c1345d4d769.jpg) and I was asked later to change the .collection-container padding-top from 15px to 10px. Theres also a bit of line height space to consider.